HW5/Plant.java

```diff
@@ -0,0 +1,33 @@
+package HW5;
+
+abstract public class Plant {
+    private class PlantHealth {
+        double waterLevel = 1.0;
```

Comment:
let's add possibility to change waterLevel from main class using new method changeWaterLevel

Reply:
added